tests/autotune_test_yamls/manifests/da/autotune_config_yaml/layer_presence_query/blank-layer-presence-query.yaml

```diff
@@ -15,8 +15,8 @@ tunables:
 - name: MaxInlineLevel
   description: 'maximum number of nested calls that are inlined. Default is 9'
   value_type: integer
-  upper_bound: "50"
-  lower_bound: "9"
+  upper_bound: 50
+  lower_bound: 9
```

Comment:
step is missing here and in other yamls that use hotspot config layer. As discussed, I think it would be good to dynamically create these yamls in results directory before the test run using the yamls in autotune/manifests. For now, I 'm ok to have these yamls updated and later include a PR.